website/src/docs/tus.md

```diff
@@ -15,8 +15,7 @@ import Tus from '@uppy/tus'
 
 uppy.use(Tus, {
   endpoint: 'https://tusd.tusdemo.net/files/', // use your tus endpoint here
-  resume: true,
-  retryDelays: [0, 1000, 3000, 5000],
+  retryDelays: [0, 1000, 3000, 5000]
```

Comment:
aren't trailing commas part of our linting rules?

Reply:
It is, that’s why I add it back (are you reviewing an old version?)

Reply:
Must be then. This somehow appeared in my notifications and i jumped on it :)  
![image](https://user-images.githubusercontent.com/26752/124129457-1819a380-da7e-11eb-9db9-a9ddf1377c1d.png)